PluginGenerator/JarManifestBuilder.cs

```diff
@@ -0,0 +1,162 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="JarManifestBuilder.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace SonarQube.Plugins
+{
+    /// <summary>
+    /// Builds a valid v1.0 Jar Manifest.
+    /// See http://docs.oracle.com/javase/6/docs/technotes/guides/jar/jar.html#JAR%20Manifest
+    /// </summary>
```

Comment:
Previously, we were calling the Java *jar* utility and it was building valid manifest files for us. Now we're manually creating them, so we're responsible for ensuring they are valid.
The main issue is with splitting long values across multiple lines.